documentation/ame-guide/modules/ROOT/pages/migration-of-models.adoc

```diff
@@ -0,0 +1,54 @@
+:page-partial:
+
+[[migration-of-models]]
+= Migration of models
+
+There are three different ways to migrate Aspect Model:
+
+* Automatic migration when starting the application.
+* Delayed migration in the sidebar.
+* Loading an older Syntactic Valid Aspect Model.
+
+[[automatic-migration]]
+== Automatic migration when starting the application
+
+As soon as the application is started and an Aspect Models with an older version is found in the workspace.
+A dialog appears with the information whether the aspect models want to be migrated.
+
+image::automatic-migration.png[width=100%]
+
+* If the checkbox Auto-increase namespace versions is activated, the Aspect Models are updated to the next higher version and the old Aspect Models are no longer displayed in the workspace.
+
+* If the checkbox Auto-increase namespace versions is not activated, the Aspect Modes will be overwritten without the version of the Aspect Models increasing.
+
+NOTE: A backup of the old Aspect Models will be created in the workspace as zip package.
+
+image::migration-status.png[width=100%]
+
+image::version-increased.png[width=100%]
+
+After the migration has been completed, the Aspect models can be modelled as usual.
+
+[[delayed-migration]]
+== Delayed migration in the sidebar
+
+If the migration of the Aspect Models in the workspace is to be carried out later, this can be cancelled for the first time.
+A migration button is provided in the sidebar under Workspace to start this process again.
```

Comment:
A migration button is provided in the sidebar below ((/within?)) the workspace to start a pending migration.